platform/kube/inject/inject.go

```diff
@@ -83,11 +83,10 @@ const (
 // Defaults values for injecting istio proxy into kubernetes
 // resources.
 const (
-	DefaultSidecarProxyUID   = int64(1337)
-	DefaultVerbosity         = 2
-	DefaultHub               = "docker.io/istio"
-	DefaultMeshConfigMapName = "meshConfig"
-	DefaultImagePullPolicy   = "IfNotPresent"
+	DefaultSidecarProxyUID = int64(1337)
+	DefaultVerbosity       = 2
```

Comment:
Lets set this to 0. Benchmarks showed a big bump in perf when logging was disabled.

Reply:
These are implicitly used by other tests. We need to set right values there first before changing defaults.